test/extended/util/util.go

```diff
@@ -144,41 +143,34 @@ func RemoveCRCHome(crcHome string) error {
 	return fmt.Errorf("folder %s not removed as per request: %s present", crcHome, keepFile)
 }
 
-// Based on the number of iterations for a given timeout in seconds the function returns the duration of echa loop
-// and the extra time in case required to complete the timeout
-func GetRetryParametersFromTimeoutInSeconds(iterations, timeout int) (time.Duration, time.Duration, error) {
-	iterationDuration, err :=
-		time.ParseDuration(strconv.Itoa(timeout/iterations) + "s")
-	if err != nil {
-		return 0, 0, err
-	}
-	extraTime := timeout % iterations
-	if extraTime != 0 {
-		extraTimeDuration, err :=
-			time.ParseDuration(strconv.Itoa(extraTime) + "s")
-		if err != nil {
-			return 0, 0, err
-		}
-		return iterationDuration, extraTimeDuration, nil
-	}
-	return iterationDuration, 0, nil
+// MatchWithRetry will execute match function with expression as arg
+// for #iterations with a timeout
+func MatchWithRetry(expression string, match func(string) error, iterations, timeoutInSeconds int) error {
+	return MatchRepetitionsWithRetry(expression, match, 1, iterations, timeoutInSeconds)
 }
 
-func MatchWithRetry(expression string, match func(string) error, retryCount, timeout int) error {
-	iterationDuration, extraDuration, err :=
-		GetRetryParametersFromTimeoutInSeconds(retryCount, timeout)
-	if err != nil {
-		return err
-	}
-	for i := 0; i < retryCount; i++ {
-		err := match(expression)
-		if err == nil {
-			return nil
+// MatchWithRetry will execute match function with expression as arg
```

Comment:
it should be `MatchRepetitionsWithRetry` instead `MatchWitRetry` for this function.

Reply:
done